packages/inventory-compliance/src/SystemPolicyCard.js

```diff
@@ -54,7 +54,7 @@ class SystemPolicyCard extends React.Component {
     }
 
     render() {
-        const { policy, benchmark, rulesPassed, rulesFailed, compliant, lastScanned, score } = this.state.policy;
+        const { policy, benchmark, _, rulesFailed, compliant, lastScanned, score } = this.state.policy;
```

Comment:
Why the `_`? If it's not used don't assign it. ;)

Reply:
Heh, I thought the policy is an array. I still have to get use to this React-fu. :-)